geometry_msgs/Point marker_enu
uint8 aruco_id
